src/commands/cicdPipeline.ts

```diff
@@ -0,0 +1,69 @@
+import * as vscode from 'vscode';
+import * as k8s from 'vscode-kubernetes-tools-api';
+import { IActionContext } from 'vscode-azureextensionui';
+import { CloudExplorerV1 } from 'vscode-kubernetes-tools-api';
+
+export async function configurePipeline(context: IActionContext, target: any): Promise<void | undefined> {
+    const deployToAzureExtensionInstalled = await isDeployToAzureExtensionInstalled();
+    if (!deployToAzureExtensionInstalled) {
+        return undefined;
+    } else {
```

Comment:
💡 Since we bail out early this else is redundant in my opinion. So remove this else and leave rest of the code as it is. Let me know if it’s clear.